src/react-apps/applications/shared/src/navigation/drawer/drawerMenuSettings.ts

```diff
@@ -4,19 +4,21 @@ export interface IMenuItem {
   menuType?: string;
   activeLeftMenuSelection?: string;
   iconName?: string;
+  [index: string]: string;
 }
 
 export interface IMainMenu {
   menuType: string;
   menuItems: IMenuItem[];
 }
 
-export interface ILeftDrawerMenu {
+export interface IDrawerMenu {
   about: IMenuItem[];
   create: IMenuItem[];
   language: IMenuItem[];
   test: IMenuItem[];
   publish: IMenuItem[];
+  [index: string]: IMenuItem[];
```

Comment:
Just being picky 😄 indexes are often numbers, so call this `index`.

Reply:
I doesn't have anything to do with the functionality.

Reply:
totally agree :) but i thought it would be easy to read and understand